### ISSUE TITLE
🐛 [BUG] - black screen on Samsung DeX

### ISSUE THREAD

**Issue by jwallet** [madmustachecompany/Blast-Workout-App#44]
### Description

Might be a bug or an improvement to support DeX.

I only see the bottom nav bar on Samsung DeX. So I cannot cast it to my tv. All React native apps load perfectly on it and electron apps and native apps as well. I wonder what UI kit you are using to make it impossible. Not a deal breaker but it was easier with my previous app to workout with my girlfriend at home.

### Reproduction steps

```bash
Connect a Samsung phone to a monitor using a usbC cable it will open in DeX
```

### Screenshots

```bash
![20230630_003832](https://github.com/madmustachecompany/Blast-Workout-App/assets/23088305/72b6c27b-20ff-4140-b20d-29cc8dc973cc)
```

### OS

Android

**Comment by alwex** [comment]
The problem has been identified and will be fixed in `1.52.x` (coming soon).

For information, the issues came from the calculation of the safe area insets which always returns `0` when the app runs in free window mode (which is the case for SamsungDex). To prevent some jump in the UI, we return a blank screen until the safe area is calculated and then replace it with the real content. 

This obviously did not work in free window mode.

Thank you for the detailed bug report, it was very helpful.

**Comment by jwallet** [comment]
Thanks I can confirm this is now fixed.
